examples/third_party/python/BUILD.python3.bazel

```diff
@@ -0,0 +1,34 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+)
+
+configure_make(
```

Comment:
For the current set of MacOS errors, maybe it's possible to build only a specific target to avoid whatever is requiring svn? I don't want macos to block this PR but I also feel that all our examples should be cross platform.

Reply:
I couldn't figure out a way to avoid the need for svnversion. I could do one of two things:

1. Run only the test for python3 on macos and omit the python2 test
2. build subversion from source and provide the binary to the python2 build. Subversion requires apr and apr-util, which will also need building from source.

I'm happy to give option 2 a go if you think it's best

Reply:
I'd be fine ignoring the py2 tests. To me the point of building python using these rules isn't to run the tests, it's just to produce a usable binary. I think in nearly all cases you'd ignore the tests and just build the targets you want maybe after running the tests against your execution or target environment once before. It might be cool to build subversion but I'm worried about that going down a rabbit hole where there are many more dependencies to build.

Reply:
I have added some commits to this PR to build subversion so that the tests can run on macos